packaging/samba.spec.j2

```diff
@@ -1132,6 +1132,11 @@ rm -f %{buildroot}%{perl_archlib}/vendor_perl/auto/Parse/Pidl/.packlist
 rm -rf %{buildroot}%{perl_vendorlib}/Parse/Yapp
 popd
 
+# packaged by libtalloc-devel
+rm -f %{buildroot}%{_mandir}/man3/talloc.3.gz
+# packaged by libldb-devel
+rm -f %{buildroot}%{_mandir}/man3/ldb.3.gz
```

Comment:
It can be that man-pages are compressed at a later step. I do not know how to access the build logs (maybe run `mock --verbose` in the CI job, or capture the logs?), so I can not easily verify this.

Reply:
This seems to be the reason: removing `.../talloc.3*` instead of `.../talloc.3.gz` lets me build the RPMs successfully. PR updated.

Reply:
Wondering why the newly added libtalloc_*gz manpages (as of e.g. libtalloc 2.3.2) are not getting in the way. Maybe because there is a rather old libtalloc-devel installed?

Reply:
not sure. they don't seem to be installed here.
Maybe because of building the libs bundled?